frontend/app/services/work-package-service.js

```diff
@@ -25,6 +25,7 @@
 //
 // See doc/COPYRIGHT.rdoc for more details.
 //++
+/* globals URI */
```

Comment:
this is a `jshint` comment with semantic function: If not used, @houndci would complain about `URI` being used without declaring it.